test/internal/big.c

```diff
@@ -0,0 +1,109 @@
+/* *
+ * Copyright (C) 2018 XLAB d.o.o.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of either:
+ *
+ *     * the GNU Lesser General Public License as published by the Free
+ *     Software Foundation; either version 3 of the License, or (at your
+ *     option) any later version.
+ *
+ * or
+ *
+ *     * the GNU General Public License as published by the Free Software
+ *     Foundation; either version 2 of the License, or (at your option) any
+ *     later version.
+ *
+ * or both in parallel, as here.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <gmp.h>
+#include <internal/big.h>
```

Comment:
Use quotation marks instead of angle brackets for including the library's headers (and maybe move it below AMCL includes).